doc: nrf: mention MMSS sample as alternative to ATv2 (#9904)
nrfconnect/sdk-nrf: doc/nrf/ug_nrf9160_gs.rst

  For the iBasis SIM card provided with the nRF9160 DK, see `iBasis IoT network coverage`_.
  If your location has support for both, you can choose either one.

  Alternatively, for a more concise nRF Cloud firmware application, you can build and install the :ref:`nrf_cloud_mqtt_multi_service` sample.

>   See :ref:`gs_programming` for information about how to build a firmware sample application.

Reviewer: ```suggestion
  See :ref:`gs_programming` for details on building a firmware sample application.
```
